lesson6/README.md

```diff
@@ -0,0 +1,87 @@
+# Lesson 6 - How to Create an App
+
+##  Designing an app
+
```

Comment:
maybe preface this with a short intro to what this lesson is about? also specify the due date of it above